smartdashboard/schemas/application.py

```diff
@@ -24,25 +24,16 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-import pytest
+import typing as t
 
-from smartdashboard.utils.helpers import get_value
+from smartdashboard.schemas.base import BaseEntity
 
-from ..utils.test_entities import *
 
-
-@pytest.mark.parametrize(
-    "key, entity, expected_value",
-    [
-        pytest.param("name", application_1, "app1"),
-        pytest.param("path", application_1, "app/1/path"),
-        pytest.param("type", orchestrator_1, "redis"),
-        pytest.param("perm_strat", ensemble_2, "all-perm"),
-        pytest.param("path", ensemble_3_member_1, "member 1 path"),
-        pytest.param("path", None, ""),
-        pytest.param("key doesn't exist", application_1, ""),
-    ],
-)
-def test_get_value(key, entity, expected_value):
-    val = get_value(key, entity)
-    assert val == expected_value
+class Application(BaseEntity):
+    path: str
+    exe_args: t.List[str] = []
+    run_settings: t.Dict[str, t.Any] = {}
+    batch_settings: t.Dict[str, t.Any] = {}
+    params: t.Dict[str, t.Any] = {}
```

Comment:
I don't think `run_settings` can be a `dict[str, str]` because `run_args` is `t.Dict[str, t.Union[int, str, float, None]]`. `params` is `t.Dict[str, t.Any]` in the SmartSim repo, are we sure it won't be anything but a string?

`batch_settings` shall be done for sure!